xbmc/cores/VideoPlayer/VideoRenderers/HwDecRender/RendererVAAPI.h

```diff
@@ -22,9 +22,16 @@
 
 #include "system.h"
 
+#ifdef HAS_GL
 #include "cores/VideoPlayer/VideoRenderers/LinuxRendererGL.h"
+using CRendererVAAPIBase = CLinuxRendererGL;
```

Comment:
that is much better than forcing constraints on base classes. assume a change in LinuxRendererGL. your code here would foce a change in LinuxRendererGLES that in turn forces changes on all of its subclasses. that is not maintainable.